src/applications/disability-benefits/2346/Components/ConfirmAddress.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getVeteranAddressInfo } from '../actions';
+
+class ConfirmAddress extends Component {
```

Comment:
We should add PropTypes to this component on the next line 🙂 It would look something like:

```jsx
static propTypes = {
  // From mapStateToProps.
  addressLine1: PropTypes.string.isRequired,
  addressLine2: PropTypes.string.isRequired,
  city: PropTypes.string.isRequired,
  state: PropTypes.string.isRequired,
  country: PropTypes.string.isRequired,
  zip: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired,
  // From mapDispatchToProps.
  getVeteranAddressInfo: PropTypes.func.isRequired,
}
```

Reply:
@mr0sari0  Please take care of this.

Reply:
@kelsonic @mojaray2k This is completed.